packages/@aws-cdk/aws-autoscaling/test/scheduled-action.test.ts

```diff
@@ -58,53 +53,40 @@ describe('scheduled action', () => {
     });
 
     // THEN
-    expect(stack).toMatch({
```

Comment:
Here is an instance where `MatchStyle.SUPERSET` is used. I came across this a few times and most of the time I could replace with `hasResourceProperties`. But here I couldn't find a way to access the `Parameters` portion of the template.

And in general, I think it would be helpful to have `templateMatches` for a subset. I initially tried to add `objectLike()` in the body of the function but then realized that `templateMatches` must match the full template.

Reply:
Ideally, we should add `hasParameters()` to the API.

For now, I think we just do `Template.fromStack(stack).toJSON().Parameters` to get the parameters section of the template.

Does that work?